js/articles.js

```diff
@@ -52,7 +52,7 @@ xmlhttp.onreadystatechange = function () {
       articlesDescriptionId.classList.add("articles-description-id");
       articlesDescriptionId.innerHTML = "more details";
       articlesDescriptionId.href =
-        "articles.html?" + data["articles"][articleIndex].id;
+        "article.html?" + data["articles"][articleIndex].id;
```

Comment:
you should find in array element whichhas the same id, not find element by index in array. it's huge error.